src/foam/support/view/TicketDetailView.js

```diff
@@ -3,41 +3,226 @@ foam.CLASS({
+      letter-spacing: 0.2px;
+      text-align: left;
+      color: #ffffff;
+      padding-left:10px;
+  }
+    ^ .Foam-Ticket-Label-Updated {
+      width: 47px;
+      height: 20px;
+      font-family: Roboto;
+      font-size: 12px;
+      font-weight: normal;
+      font-style: normal;
+      font-stretch: normal;
+      line-height: 1.67;
+      letter-spacing: 0.2px;
+      text-align: left;
+      color: #ffffff;
+      padding-left:10px;
+  }
+    ^ .Foam-Ticket-Label-Open {
+      width: 29px;
+      height: 20px;
+      font-family: Roboto;
+      font-size: 12px;
+      font-weight: normal;
+      font-style: normal;
+      font-stretch: normal;
+      line-height: 1.67;
+      letter-spacing: 0.2px;
+      text-align: left;
+      color: #ffffff;
+      padding-left:10px;
+  }
+    ^ .Foam-Ticket-Label-Pending {  
+      width: 45px;
+      height: 20px;
+      font-family: Roboto;
+      font-size: 12px;
+      font-weight: normal;
+      font-style: normal;
+      font-stretch: normal;
+      line-height: 1.67;
+      letter-spacing: 0.2px;
+      text-align: left;
+      color: #ffffff;
+      padding-left:10px;
+  }
+    ^ .Foam-Ticket-Label-Solved {
+      width: 37px;
+      height: 20px;
+      font-family: Roboto;
+      font-size: 12px;
+      font-weight: normal;
+      font-style: normal;
+      font-stretch: normal;
+      line-height: 1.67;
+      letter-spacing: 0.2px;
+      text-align: left;
+      color: #ffffff;
+      padding-left:10px;
+  }
+    ^ .main {
+      width: 488px;
+      height: 16px;
+      opacity: 0.7;
+      font-family: Roboto;
+      font-size: 12px;
+      font-weight: normal;
+      font-style: normal;
+      font-stretch: normal;
+      line-height: 1.33;
+      letter-spacing: 0.2px;
+      text-align: left;
+      color: #093649;
+  }
   `,
 
   methods: [
     function initE(){
-      var self = this;
-      
-      console.log(this.data)
-      this.addClass(this.myClass())
-      .start(this.BACK_ACTION).end()
-      // note to ANUBHAV, ticket object is stored in this.data
-      .start().add(this.data.id).end()
-      .start().add(this.data.createdAt).end()
-      .add(this.data.status)
+    var self = this;
+    var email=this.data.supportEmail;
+    this.userDAO.where(this.EQ(this.User.EMAIL, email)).select().then(function(a){
+      self.name=a.a[0].firstName;
+     })
+     var locale = "en-us";
+     var month = this.data.createdAt.toLocaleString(locale, {month: "short"});
+     var date=this.data.createdAt.getDate();
+     var hours=this.data.createdAt.getHours(); 
+     var mins= this.data.createdAt.getMinutes()
+       this.addClass(this.myClass())
+       .start(this.BACK_ACTION).end()
+       .start("br").end()
+       .start("br").end()
+       .start("br").end()
+       .start().addClass('primarydiv')
+            .start().addClass('Missing-Cash-Out-for').add(this.data.subject+"...").end()
+            .start().addClass('generic-status Foam-Ticket-Status-'+this.data.status)
```

Comment:
rmv `Foam-Ticket-Status-'` and leave this.data.status in addClass(). Please remove associated css classes stated in this view aswell. (.Foam-Ticket classes). This relates to my previous comment.